### ISSUE TITLE
custom fields for contact association do not display in web form

### ISSUE THREAD

**Issue by bgoulet00** [nautobot/nautobot#6475]
### Environment
* Nautobot version (Docker tag too if applicable): 2.3.6, 2.3.10, no docker
* Python version: 3.8
* Database platform, version: PSQL12
* Middleware(s):

### Steps to Reproduce
1. create custom field
```
name: myfield
content types: extras | contact association
required: checked
```

2. navigate to a location and click add contact
note that the custom field does not display in the form
complete the form and click assign.  notice the contact is not added because the custom field is required but cannot be set

this is also the case for adding contacts to circuits and most likely any object that allows contact associations

### Expected Behavior
custom field shows up in contact associations form

### Observed Behavior
custom field does not show up in contact associations form

**Comment by glennmatthews** [comment]
Thanks for the report! Looks like fixing this *might* be as simple as adding `{% include 'inc/extras_features_edit_form_fields.html' %}` to `templates/extras/object_assign_contact_or_team.html`?

**Comment by bgoulet00** [comment]
was that a question for me?  i'm not familiar with under the hood

**Comment by glennmatthews** [comment]
No, sorry for the confusion - just a note to whoever picks this up to fix it. :-)
